jambalya.md

```diff
@@ -0,0 +1,51 @@
+# Jambalya
+
+## Ingredients
+
+**meat**
+
+* 1 1/4 lb - pork
+* 1 1/24 lb - smoked sausage
+
+**veggies**
+
+* 2 - medium onions
+* 1 - bell pepper
+* 2 stalks - celery
+* 1 handful - green onions
+* 1 handful - parsley
+
+**seasoning**
+
+* 1 tbsp - chopped garlic 
+  * or 1 1/2 tsp - garlic powder
+* 2 tsp - Morton Season-All seasoned salt
+* 2 tsp - Tony Chachere's Original Creole Seasoning
+* 1/2 tsp - black pepper
+* 2 tsp - salt
+* 1/4 cup - worcestershire sauce 
+* 1 tbsp - hot sauce
+
+**liquid**
+
+* 3 1/2 cup - water 
+* 2 cups - beef stock
+
+**optional**
+
+* 1 tsp - kitchen bouquet for color
+
+## How To
+
+The critical thing to get right here is the amount of liquid in the pot when the rice goes into cook.
+
+1.  Brown pork and remove from pot 
+2.  Brown sausage and remove from pot, drain most of the grease
+3.  Cook onions till caramelized
+4.  Add the rest of the vegetables and cook till tender
+5.  Add the meat and seasoning to the pot, add just enough water to the pot so you can stir.  Note this water is not from the 3 1/2 cups in the ingredients list.  Cook (simmer?) for around 30 minutes.  "You should be able to tell by the way it looks"
+6.  Mix beef broth and water; then add to pot
+7.  Bring pot to a boil 
+8.  Add rice and cook till most of the water is gone.  About 10 to 12 minute, stirring continuously on medium high heat.  The spoon should be able to lay on top of the jambalaya  without sinking.
+9.  Reduce to very low heat and cover pot.  DO NOT REMOVE cover for 15 minutes.
```

Comment:
Want to make a note about the lowest setting you have?

Reply:
There were also some kind of notes about the quality of the seal, and coping with a poor one, I think?